app/assets/javascripts/admin/forms/project_step_form.js.coffee

```diff
@@ -0,0 +1,151 @@
+$ ->
+  formatTypeOptions = (option) ->
```

Comment:
Need to standardize JS in this project in terms of variable naming. There is a mix of snake_case and camelCase. @linesarefuzzy ?